tests/dispense.rs

```diff
@@ -200,26 +228,32 @@ async fn _dispense_sends_coins_to_valid_address(
 #[tokio::test]
 async fn many_concurrent_requests() {
```

Comment:
This test should remain almost the same. We need to send 30 transactions and be sure that all of them were committed. The balance of the addresses is optional to check, but you still can do that=)

Reply:
Problem is, the dispense limit now prevents those txs from being committed. However, If we need 30 transactions we can use 30 different addresses if that is acceptable.